ethereum/test/pyth.js

```diff
@@ -362,6 +362,60 @@ contract("Pyth", function () {
             );
         }
     });
+
+    it("should set chain/emitter/wormhole IDs and immediately use them", async function () {
+        let attestationTime = 1647273460; // re-used for publishTime
+        let publishTime = 1647273465; // re-used for publishTime
+        let rawBatch = generateRawBatchAttestation(attestationTime, publishTime, 1337);
+
+        // Switch only chain ID to wrong value
+        await this.pythProxy.setContractConfig(testPyth2WormholeChainId + 1, testPyth2WormholeEmitter, (await Wormhole.deployed()).address);
+
+        let wrongChainIdFails = false;
+
+        try {
+            // Should result in expected failure
+            await updatePriceFeeds(this.pythProxy, [rawBatch]);
+        } catch (e) {
+            wrongChainIdFails = true;
+        }
+
+        assert.equal(wrongChainIdFails, true);
```

Comment:
I'm not sure what is the assert library here, but don't we have `assert.throws` or sth similar to make it more clear?